### PR TITLE
feat: support starlight

### PR DESCRIPTION
**Pull request by liruifengv**
## What kind of changes does this PR include?

Add support Starlight

## Description

[Starlight](https://github.com/withastro/starlight) is a documentation website framework for [Astro](https://astro.build/).

Added support for displaying its logo when deployed on Zeabur.

## Related
- https://github.com/zeabur/service-icons/pull/7

### CODE REVIEW

File: zbpack
Comment (pan93412) [inline zbpack:1]: It is better to remove this binary file and squash the commit to ensure that there are no instances of this file in the entire commit history.

**Review by pan93412 — CHANGES_REQUESTED** [review]

File: zbpack
Comment (liruifengv) [reply zbpack:1]: I deleted it. Thanks.

Why not add it to `.gitignore`?

Comment (pan93412) [reply zbpack:1]: > I deleted it. Thanks.
> 
> 
> 
> Why not add it to `.gitignore`?

We just didn't do that. Feel free to open a PR for this ;)

**Review by pan93412 — APPROVED** [review]
(For merger: **Squash Merge** it!)

**Pull request merged**